doc/conf.py

```diff
@@ -37,7 +40,7 @@ def _LatestTagName():
 
 # The full version, including alpha/beta/rc tags
 version = "0.3"     # The short X.Y version.
-release = "0.3.0"   # The full version, including alpha/beta/rc tags.
+release = "0.3.1"   # The full version, including alpha/beta/rc tags.
```

Comment:
Should use newest technique from e.g. pyTooling to read version from `__init__.py`

Reply:
I think that is not prioritary. We can wait until https://github.com/pyTooling/pyTooling.Packaging is usable.

Reply:
ok